autodock_core/scripts/autodock_core/autodock_server.py

```diff
@@ -336,6 +344,19 @@ def rotate_with_odom(self, rotate: float) -> bool:
             rospy.sleep(self.sleep_period)
         exit(0)
 
+    def set_aruco_detections(self, detection_state) -> bool:
+        """
+        Set aruco detections to True or False
+        :return : success
+        """
+        try:
+            rospy.wait_for_service('/enable_detections', timeout=3.0)
+            resp = self.__enable_detections_srv(detection_state)
+            print("Enable detections response: ", resp.message)
+            return resp.success
+        except rospy.ServiceException as e:
+            print("Service call failed: %s"%e)
```

Comment:
Nit, use `rospy.logerr()` here . similarly, use `rospy.loginfo` for the print above